src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -674,6 +674,13 @@ public virtual bool SupportsBreakpointChecksums()
             return false;
         }
 
+        public virtual Task<bool> RequiresOnKeywordForBreakInsert()
+        {
+            return Task<bool>.FromResult(false);
+        }
+
+
+        public abstract Task<string> Version();
```

Comment:
Is there a reason to put either of these in the base class? Seems like we only care about these within the LLDB command factory, so they should just be private methods there.